### PR TITLE
Fixes Banner styling

### PR DESCRIPTION
**Pull request by kierangillen**
## Problem: 

screen shot 2019-01-11 at 9 42 09 am

## Fix:

screen shot 2019-01-11 at 10 03 05 am

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit b296298b4f7702513b769261976267fcf2e0fda9

https://deploy-preview-163--artsy-palette.netlify.com

File: src/elements/Banner/Banner.tsx [hunk 1]
```diff
@@ -61,19 +57,12 @@ export class Banner extends React.Component<BannerProps> {
...
-        bg={this.props.backgroundColor}
-        color="white100"
-        p={1}
-        position="relative"
-        pr={paddingRight}
-        textAlign={textAlignment}
-      >
-        <Sans size="2">{this.props.message}</Sans>
+      <Wrapper bg={this.props.backgroundColor} color="white100" p={1}>
+        <TextWrapper>
```
Comment (damassi) [inline src/elements/Banner/Banner.tsx]: This `TextWrapper` component can completely go away and width / centering can be passed into the wrapper

Comment (damassi) [inline src/elements/Banner/Banner.tsx]: This `TextWrapper` component can completely go away and the width / centering can be passed into the outer `Wrapper`

**Comment by kierangillen** [comment]
Github is acting crazy closing this PR and opening an identical one

**Comment by damassi** [comment]
👍 👍

**Pull request merged**